test/dynamo/test_subclasses.py

```diff
@@ -2009,7 +2009,7 @@ def f(nt):
             norm_graph,
             """\
 class GraphModule(torch.nn.Module):
-    def forward(self, L_nt_: "f64[3, s1, 5]", s1: "Sym(s1)"):
+    def forward(self, s1: "Sym(s1)", L_nt_: "f64[3, s1, 5]"):
```

Comment:
Why does this order end up changing?

Reply:
The logic for lifting symbols is unified to first lift the tensor then insert the symints inside it before the tensor. I think previously this ordering is not enforced globally.